evaluation/human_evaluation/main.py

```
@@ -14,16 +14,25 @@
 def main() -> None:
     load_dotenv()
 
-    GOOGLE_SHEET_ID = os.getenv("GOOGLE_SHEET_ID")
-    GOOGLE_FORM_ID = os.getenv("GOOGLE_FORM_ID")
+    google_sheet_id = os.getenv("GOOGLE_SHEET_ID")
```

Should it error out if the env var is not specified?